GANDLF/cli/deploy.py

```diff
@@ -261,12 +231,95 @@ def deploy_docker_mlcube(modeldir, config, outputdir, mlcubedir, requires_gpu):
         f"To run this container as an MLCube, you (or the end-user) should invoke the MLCube runner with --mlcube={outputdir} ."
     )
     print(
-        "Otherwise, it will function as a standard docker image of GaNDLF, but with embedded model/config files."
+        "Otherwise, it will function as a standard docker image of GaNDLF, but with possibly embedded model/config files."
     )
     print("Deployment finished successfully!")
     return True
 
 
+def get_metrics_mlcube_config(mlcube_config_file, entrypoint_script):
+    mlcube_config = None
+    with open(mlcube_config_file, "r") as f:
+        mlcube_config = yaml.safe_load(f)
+    if entrypoint_script:
+        # modify the entrypoint to run a custom script
+        mlcube_config["tasks"]["evaluate"]["entrypoint"] = "python3.8 /entrypoint.py"
```

Comment:
I will see if I can find a better way. `python3` will be version `3.6` since it will refer to the default python3 executable of the docker image's OS that is ubuntu 18.04

Reply:
Yup, this is exactly right. We could switch the base image to a more minimal image but then we'll have to probably do some more dependency installation.